TestStack.FluentMvcTesting/ControllerResultTest/ShouldRedirectTo.cs

```diff
@@ -117,7 +117,10 @@ public RouteValueDictionary ShouldRedirectTo(MethodInfo method, RouteValueDictio
             var redirectResult = (RedirectToRouteResult)ActionResult;
 
             if (redirectResult.RouteValues["Controller"] == null)
```

Comment:
Not sure off-hand. 

I will spend some time to understand the code more deeply this afternoon and then issue another commit.

Thank you.